### PR TITLE
Display common/uncommon/rare on mei variant summary

### LINKED ISSUE #4464: MEI summary frequency badge could be populated
We do show frequency info in the Frequency section, so just a matter of condensing.
Screenshot 2024-02-27 at 12 55 46

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Fix #4464 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. On demo case, go to mei variantS page (mobile elements). Pick one which has frequencies
2. Go to the variant page and make sure that the frequency common/uncommon/rare is populated on variant smmary
3. On main branch this value is empty

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested locally:

image

File: scout/server/blueprints/variant/utils.py [hunk 1]
```diff
@@ -453,6 +453,8 @@ def frequency(variant_obj):
         variant_obj.get("exac_frequency") or 0,
         variant_obj.get("gnomad_frequency") or 0,
         variant_obj.get("gnomad_mt_homoplasmic_frequency") or 0,
+        variant_obj.get("swegen_alu") or 0,
```
Comment (dnil) [inline scout/server/blueprints/variant/utils.py:456]:
```suggestion
```

Comment (dnil) [reply scout/server/blueprints/variant/utils.py:456]: If you want them, it should be all of `alu`, `sva`, `l1` and `herv`, but as long as the mei_max works, there should be no reason to have any of the individual ones!

File: scout/server/blueprints/variant/controllers.py [hunk 2]
```diff
@@ -293,7 +295,7 @@ def variant(
     # Add general variant links
     variant_obj.update(get_variant_links(institute_obj, variant_obj, int(genome_build)))
     variant_obj["frequencies"] = frequencies(variant_obj)
-    if variant_category in ["snv", "cancer"]:
+    if variant_category in ["snv", "cancer", "mei"]:
```
Comment (dnil) [inline scout/server/blueprints/variant/controllers.py:298]: 👍 Good call - that means we get the same limits as for the SNVs.

**Review by dnil — APPROVED** [review]
Great! There should be no need for the individual alu-freq though.

**Pull request merged**